system-variables.md

```diff
@@ -2025,6 +2047,28 @@ For details, see [Identify Slow Queries](/identify-slow-queries.md).
 - Range: `[0, 9223372036854775807]`
 - This variable is used to limit the maximum number of requests TiDB can send to TiKV at the same time. 0 means no limit.
 
+### tidb_super_read_only <span class="version-mark">New in v5.3.1</span>
+
+> **Warning:**
+>
+> For TiDB versions earlier than v5.3.1 or v5.4.1, this variable has a defect and using it might cause unexpected results. Ensure that you only use this variable in TiDB versions later than v5.3.1 or v5.4.1.
```

Comment:
```suggestion
> For TiDB versions earlier than v5.3.1 or v5.4.1, this variable has a defect and using it might cause unexpected results. Ensure that you use this variable ONLY in TiDB versions later than v5.3.1 or v5.4.1.
```